locale/footer/en.yaml

```diff
@@ -5,6 +5,9 @@ links:
     manualLink:
         href: 'http://manual.zetkin.org/en'
         text: The Zetkin Manual
+    privacyLink:
+        href: 'http://zetkin.org/privacy'
+        text: The Zetkin Privacy Policy
```

Comment:
Why the long name? Akin to the Swedish string, I prefer the shorter, simpler "Privacy Policy".